manifests/init.pp

```diff
@@ -38,7 +38,8 @@
   $file_group                         = $bareos::params::file_group,
   $file_mode                          = $bareos::params::file_mode,
   $file_dir_mode                      = $bareos::params::file_dir_mode,
-  String  $repo_release               = '21',
+  $user_groups                        = $bareos::params::user_groups,
+  String  $repo_release               = '22',
```

Comment:
Changing the default version make it a breaking change. This is just a FYI, if the latest version is 22 we can change the default in the next (major) version. I will add a backward-incompatible label accordingly.

Reply:
I see #161 is about adding support for bareos 23. Do you plan to rebase this version too next?